draft/draft-ietf-bfd-optimizing-authentication.xml

```diff
@@ -904,16 +999,30 @@ reference:    RFC XXXX
 	solution to update BFD Authentication that is currently specified in
 	<xref target="RFC5880"/>.  This experiment is intended to
 	provide additional insights into what happens when the
-	optimized authentication method defined in this document is
+	optimized authentication mechanism defined in this document is
 	used. Here are the reasons why this document is on the Experimental track:
       <t><list style="symbols">
-	      <t>In the initial stages of the document, there were significant participation and reviews from the working group. 
-	Since then, there has been considerable changes to the document, e.g. the use of ISAAC, allowing for ISAAC bootstrapping 
-	when a BFD session comes up and use of a single Auth Type to indicate use of optimized authentication etc. 
-	These changes did not get significant review from the working group and therefore does not meet the bar set in Section 4.1.1 of <xref target="RFC2026"/></t>
-	<t>There are no known implementations (even proof-of-concept) or implementation plans. As a result, we do not currently know if there will be interop issues with 
-	legacy implementations or what exactly are the performance benefits of the optimization method.</t>
-	<t>The work in this document could become very valuable in the future, especially if the need for deploying BFD authentication at scale becomes a reality.</t>
+        <t>
+	  In the initial stages of the document, there were significant
+	  participation and reviews from the working group.  Since then, there
+	  has been considerable changes to the document, e.g. the use of ISAAC,
+	  allowing for ISAAC bootstrapping when a BFD session comes up and use
+	  of a single Auth Type to indicate use of optimized authentication
+	  etc.  These changes did not get significant review from the working
+	  group and therefore does not meet the bar set in Section 4.1.1 of
+	  <xref target="RFC2026"/>
+	</t>
+	<t>
+	  There are no known implementations (even proof-of-concept) or
+	  implementation plans. As a result, we do not currently know if there
```

Comment:
Can we drop "or implementation plans"? It is difficult to predict, first of all, that will be no implementations, and secondly if and when somebody might decide to do an implementation. This also caused all the ABSTAINS in IESG review.